### DISCUSSION TITLE
The route in the map is a straight line

### DISCUSSION THREAD

**Discussion by zhoubingyi** [teslamate-org/teslamate#3318] [Q&A]
![image](https://github.com/adriankumpf/teslamate/assets/22286641/0522df25-e84d-48ae-a0ac-f3ecb9b8d67c)

When the geographic location is collected, the places where the car is located are connected in a straight line, and the drawn trajectory map does not coincide with the road

**Comment by cwanja** [comment]
That is due to the API not providing data fast enough as you are driving. So the positions are logged every five seconds as opposed to every second. As such, when the map makes the route based off position data, it is not a smooth line that follows the road. 

I have some drives that are like this and others that are smooth.
